pyhoma/client.py

```diff
@@ -206,23 +268,24 @@ async def check_response(response: ClientResponse) -> None:
         #  'error': 'Too many requests, try again later : login with xxx@xxx.tld'}
         #  'error': 'Bad credentials'}
         #  'error': 'Your setup cannot be accessed through this application'}
-        result = await response.json()
-        if response.status == 401:
-            if result.get("errorCode") == "AUTHENTICATION_ERROR":
-                message = result.get("error")
-
-                if "Too many requests" in message:
-                    raise TooManyRequestsException(message)
+        try:
+            result = await response.json(content_type=None)
+        except JSONDecodeError:
+            body = await response.text()
+            raise Exception(
+                f"Something bad happened when requesting {response.url}. {body}"
```

Comment:
Will this actually fix #35? Or was the fix to add `content_type=None`, since the TaHoma API sometimes adds the wrong mimetype?

Reply:
The one fixing #35 is `content_type=None`. It avoid aiohttp to look for the mimetype. It's why now I have to check for JSONDecodeError in case JSON is not returned.